addons/sale/res_config_view.xml

```diff
@@ -70,6 +82,25 @@
                         </div>
                     </group>
                 </group>
+                <group name="Sale" position="after">
+                    <group name="E-Commerce Features">
+                        <label for="id" string="E-Commerce Features"/>
```

Comment:
Hello @jem-odoo  yes label is required here. string attribute with group tag didnot work.